services/kommander-flux/2.1.0/templates/apiextensions.k8s.io_v1_customresourcedefinition_helmrepositories.source.toolkit.fluxcd.io.yaml

```diff
@@ -327,8 +346,9 @@ spec:
               secretRef:
                 description: SecretRef specifies the Secret containing authentication
                   credentials for the HelmRepository. For HTTP/S basic auth the secret
-                  must contain 'username' and 'password' fields. For TLS the secret
-                  must contain a 'certFile' and 'keyFile', and/or 'caFile' fields.
+                  must contain 'username' and 'password' fields. Support for TLS auth
+                  using the 'certFile' and 'keyFile', and/or 'caFile' keys is deprecated.
+                  Please use `.spec.certSecretRef` instead.
```

Comment:
i think we use `secretRef` for airgapped, should we take note of this deprecation? it sounds like we don't need to handle it now but maybe a ticket for the future so we don't fall behind.

Reply:
good point! from [kcli code here](https://github.com/mesosphere/kommander-cli/blob/main/pkg/upgrade/app_repo_updater.go#L282) and [kommander code here](https://github.com/mesosphere/kommander/blob/main/federation/cmd/appmanagement/main.go#L164-L165) seems like we are just using the HTTP basic auth. Guess we are good?

Reply:
ah missed that lol 🙈 . yeah could you make a ticket for that? @cbuto

Reply:
yep created one - https://d2iq.atlassian.net/browse/D2IQ-99228